cspell.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Why was this file included? 🤔

Reply:
That one slipped through I forgot to delete it. It's for the code spellchecker

Reply:
No worries. Our team doesn't use a spellchecker (as you might have guessed 😅) but this actually seems interesting, will consider it! It's [this one](https://cspell.org/), right?

Reply:
I went ahead and deleted the file 🙂